packages/perseus/src/widgets/expression.tsx

```diff
@@ -35,28 +35,6 @@ type InputPath = ReadonlyArray<string>;
 const ERROR_TITLE = i18n._("Oops!");
 const ERROR_MESSAGE = i18n._("Sorry, I don't understand that!");
 
-const insertBraces = (value) => {
```

Comment:
We were able to remove this because the new fork always returns braces around super/subscript!